packages/runtime/src/integrations.ts

```diff
@@ -61,7 +61,8 @@ export function createIntegration<Context extends RuntimeContext = RuntimeContex
             const event = JSON.parse(formData.get('event') as string) as Event;
             const fetchBody = formData.get('fetch-body');
             const context = createContext(
-                JSON.parse(formData.get('environment') as string) as IntegrationEnvironment
+                JSON.parse(formData.get('environment') as string) as IntegrationEnvironment,
+                ev
```

Comment:
Don't put it on the context, prefer passing it directly to the handler on line 78:

```
await definition.fetch(request, ev, context)
```

This keeps a cleaner scope between a context (potentially long-lived across multiple requests) and a request/FetchEvent (different per request)

Reply:
shouldn't this be  `await definition.fetch(request, context, ev)` to prevent breaking changes to current integrations?

Reply:
aha I've read the Slack thread and I'm more up-to-date now.

In that case I'd recommend as you've done using the context but just passing the `waitUntil` function, so calling code could do: `context.waitUntil(() => {})`

Reply:
Main reason is we want to avoid something doing `context.ev.request.url` or something

Reply:
ok, updated!